README.md

```diff
@@ -1,26 +1,56 @@
 
-# About
+# lsif-util
 
-Scripts to help [LSIF](https://github.com/Microsoft/language-server-protocol/blob/master/indexFormat/specification.md) developers. Featuring:
+Scripts to help [LSIF](https://github.com/Microsoft/language-server-protocol/blob/master/indexFormat/specification.md) developers.
 
-* Validation
-* Graph visualization
-* Quick search
+## What's new
 
-# Getting Started
+* _Up to date!_ Support for [JSON Lines](http://jsonlines.org/) with the `--inputFormat` tag
+* _Cleaning up!_ Standard filtering for all tools (no more unnecessary searching)
 
-* `> git clone this repository`
-* `> cd lsif-util`
-* `> npm install`
-* `> npm run compile`
+## What's next
 
-# Validation
+* _We are moving!_ The lsif-util tools will soon migrate to [lsif-node](https://github.com/microsoft/lsif-node)
+* _Getting official!_ Global npm package is on the way
 
-`> node .\lib\validate.js [options]`
+## Usage
 
-| Option            | Default     | Description                                             |
-|-------------------|-------------|---------------------------------------------------------|
-| --inputPath or -p | ./lsif.json | Path to input file (JSON)                               |
+``` bash
+git clone https://github.com/jumattos/lsif-util.git
+cd lsif-util
+npm install
+npm run compile
+node .\lib\main.js [validate|visualize] [file] --inputFormat [line|json] [--stdin] [filters]
+```
+
+| Option        | Description                                    | Default |
```

Comment:
In my opinion, this list can be a bit intimidating for a new user. I'd like to discuss how we can make it easier to understand. Some suggestions

- can we break it into mandatory and optional parameters?
- for the filtering parameters, we could create a separate heading that shows how filtering works. In that section, we can put headings like "visualizing only the document vertices", and show the relevant filter that would be applicable.
- examples for parameters would be useful. It's not clear, for example, whether id is [1,2] or ["1","2"]

Related to the last point: generally, CLI tools seem to use comma separated values when inputting a list of them, without the square brackets. Will users have to specify the square brackets?

Reply:
I wrote a new filter table that might be more descriptive (I hope so!). It should answer your question and hopefully clear any confusion: https://github.com/jumattos/lsif-util/tree/update-ui

Reply:
Looks good. Thanks!